.jenkins/gpuarray-branch

```diff
@@ -0,0 +1 @@
+v0.6.9
```

Comment:
I think we should lock to a specific tag or even release. Because, in the end Theano is not meant to work with any possible state of the master of libgpuarray, and we have had too much breakage in the past because the master introduced incompatible changes.

The buildbots still test master against master, even though I think they shouldn't either.